timer/runtime/worker.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	workerRecvChanCap               = 8
+	workerRecvChanCap               = 128
```

Comment:
Any reason to enlarge this?

Reply:
To make the timer trigger more smoothness when we have many timers in one hook (.i.e 1000 ttl tables)